Rewrite "composite approaches" section (#139)
jaybee84/ml-in-rd: content/06.multiple-approaches-required.md

### Using composite approaches can be a powerful strategy

We have described multiple approaches for maximizing the success of machine learning applications in rare disease research throughout. 
In practice, it is rarely sufficient to use one of these techniques in isolation. 
Below, we highlight two recent works in the rare disease domain that draw on multiple concepts covered in the earlier sections. 
Feature-representation-transfer, which incorporates dimension reduction through representation learning, prior data, and regularization underlie both approaches.

Thousands of acute myeloid leukemia (AML) patient gene expression samples have been collected over time and are publicly available.
However, as is often the case with publicly available data, not all of these samples include the most relevant clinical or phenotypic data such as drug response.

> The authors of Dincer et al. had access to an _in vitro_ experiment that examined the response to 160 drugs for 30 AML patient samples, measured using genome-wide arrays which have tens of thousands of features [@doi:10.1038/s41467-017-02465-5].

Reviewer: ```suggestion
```

Training on rare disease datasets with a low number of samples, a considerable number of tasks, and many features can result in ML models that are of limited utility.

> The low number of samples in the drug response dataset could also be prohibitive when performing representation learning.

Author: If taking the above suggestion, this sentence would also need to be moved later in my opinion.

> Instead, the authors trained a variational autoencoder on over 6500 AML samples (VAE; see [definitions]) [TODO: link between sections?] to reduce the dimensionality of the test set in an approach termed DeepProfile [@doi:10.1101/278739v2].

Reviewer: ```suggestion
In view of these challenges, an _in vitro_ experiment that examined the response to 160 drugs for 30 AML patient samples, measured using genome-wide arrays that have tens of thousands of features was capitalized upon [@doi:10.1038/s41467-017-02465-5].
The authors trained a variational autoencoder on over 6500 AML samples (VAE; see [definitions]) [TODO: link between sections?] to reduce the dimensionality of the test set in an approach termed DeepProfile [@doi:10.1101/278739v2].
```

The 30 AML test samples with drug response information were encoded using the VAE's low-dimensional representation or _transferred_, reducing the number of features from thousands to 8.
LASSO linear regression models that used encodings as features had better performance than models that used individual gene expression values as features on average.
In addition, the low-dimensional representation learned by the VAE captured more biological pathways than PCA, which may be attributable to the constraints on encodings imposed during the training process [definitions] [TODO: link between sections or cite what is in definitions?].
Similar results were observed for prediction of histopathology in another rare cancer (ovarian).

While DeepProfile was centered on training on an individual disease and tissue combination, some rare diseases affect multiple tissues that a researcher may be interested in studying together for the purpose of biological discovery. 

> Studying multiple tissues poses challenges – features that can be measured an appreciable level may be tissue-specific even when looking at learned features or representations, cross-tissue analysis may require an analyst to compare representations from multiple models and models trained on a low number of samples may learn representations that "lump together" multiple biological signals, reducing the interpretability of the results.

Reviewer: ```suggestion
Studying multiple tissues poses significant challenges – features that can be appreciably measured may be tissue-specific even when looking at learned features or representations, a cross-tissue analysis may require an analyst to compare representations from multiple models and models trained on a low number of samples may learn representations that "lump together" multiple biological signals, reducing the interpretability of the results.
```

> To address these challenges, Taroni et al. trained Pathway-Level Information ExtractoR (PLIER) [@doi:10.1038/s41592-019-0456-1] on a large generic collection of human transcriptomic data (recount2 [@doi:10.1038/nbt.3838]) and used the latent variables learned by the model to describe transcriptomic data from the unseen rare diseases antineutrophil cytoplasmic antibody (ANCA)-associated vasculitis (AAV) and medulloblastoma in an approach termed MultiPLIER [@doi:10.1016/j.cels.2019.04.003]. 

Reviewer: ```suggestion
To address these challenges, a Pathway-Level Information ExtractoR (PLIER) [@doi:10.1038/s41592-019-0456-1] was trained on a large generic collection of human transcriptomic data (recount2 [@doi:10.1038/nbt.3838]).
The latent variables learned by the model were used to describe the transcriptomic data from unseen rare diseases antineutrophil cytoplasmic antibody (ANCA)-associated vasculitis (AAV) and medulloblastoma in an approach termed MultiPLIER [@doi:10.1016/j.cels.2019.04.003]. 
```
